game/src/js/objects/FireMonster.js

```diff
@@ -0,0 +1,12 @@
+import Character from './Character';
+
+export default class FireMonster extends Character {
```

Comment:
We removed these Character subclasses and are now using the `Character` class directly configured from JSON.  Please use the `Character` class directly.

Reply:
I know that, I just couldn't figure out how to make it work that way.  I can try again though.